packages/monaco/src/split-editor.tsx

```diff
@@ -245,7 +248,10 @@ export default function SplitEditor({
           defaultPath={TESTS_PATH}
           value={tests}
           defaultValue={tests}
-          onChange={onChange?.tests}
+          onChange={async (e, a) => {
+            console.log({ e, a });
```

Comment:
ayo 2

Reply:
this PR aint ready chill lmaooo shoulda did draft

Reply:
yeah I'm teaching you the way to draft 

![image](https://github.com/typehero/typehero/assets/996134/6d34e696-326c-4003-baf1-e92fcda2a1d1)